db/schema.rb

```diff
@@ -90,9 +90,6 @@
     t.index ["code"], name: "index_courses_on_code", unique: true
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
-  end
```

Comment:
Should this be removed?

Reply:
Not sure what's happened there 🤔

Reply:
I recreated the db locally and this still gets removed.
I wonder if it got added by someone in their local db, there's no create migration for this.

Reply:
This PR added the gem `data_migrate` https://github.com/DFE-Digital/register-trainee-teachers/pull/873 - it's now in master.

When any of the rake tasks are run e.g. `rake data:migrate` then a `data_migrations` table is created.